### DISCUSSION TITLE
Basic bootstrap / install instructions?

### DISCUSSION THREAD

**Discussion by tetsuosoft** [heckj/Lindenmayer#35] [Q&A] [answered]
Hello @heckj and folks

This looks like a fine package, unfortunately I cant seem to get it up and running to play with it

I've imported the package into a vanilla SwiftUI app, etc. but whenever I try to call a struct in my default ContentView eg. Lsystem2DView, I cant get past the pre-compile error "Cannot find 'Lsystem2DView' in scope"

Any tips?

Cheers!

**Comment by heckj** [comment, marked as answer]
hi @roboldham,

When I set up the package, I separated the core bits that made up the L-systems from the views that can present them. So in addition to doing `import Lindenmayer`, if you want to use the views, you'll also need to `import LindenmayerViews`, which is where the swiftUI views for exploring and experimenting are stashed. It's a separate product included in the same swift package.

It's separated out to try and keep the elements you need minimal to the platform and goal you're after. For example, the SwiftUI views might not even be useful if you're fiddling with ARKit.

**Comment by tetsuosoft** [comment]
Ahh... I shall go and put my dunces cap on for the evening! Thanks, that was simple! Gonna go have some fun now! Thank you

**Reply by heckj** [reply]
No worries. I've been primarily using the LindenmayerViews target as a means to explore and experiment while I get the rest set up - there's ARKit support at the moment, but I haven't evolved things forward (yet) to support RealityKit, which is a longer term goal. I've been a bit distracted by another project recently sucking up my time, but I expect WWDC will revitalize (and perhaps evolve) a lot of the pattern of how I enable other 3D-oriented Lindenmayer support.

Quick example to get you started, since I don't have this written down anywhere yet:

```swift
import SwiftUI
import Lindenmayer
import LindenmayerViews

struct ContentView: View {
    var body: some View {
        Lsystem2DView(system: Lindenmayer.Examples2D.kochCurve.evolved(iterations: 5))
    }
}

struct ContentView_Previews: PreviewProvider {
    static var previews: some View {
        ContentView()
    }
}
```
<img width="1378" alt="Screen Shot 2022-05-29 at 3 12 15 PM" src="https://user-images.githubusercontent.com/43388/170893297-ceda7e5f-5377-4494-8358-8a77a0fd3245.png">

**Reply by tetsuosoft** [reply]
Haha thanks, weirdly that's almost exactly what I did to the character, to get strapped in
